api/v1beta1/cluster_types.go

```diff
@@ -332,6 +332,11 @@ type Topology struct {
 	// The name of the ClusterClass object to create the topology.
 	Class string `json:"class"`
 
+	// The namespace of the ClusterClass object to create the topology.
```

Comment:
What is the behaviour when this is not specified? Can you add that to the description here?